pharo/run.sh

```diff
@@ -124,7 +124,10 @@ moose::get_image_url() {
       echo "https://github.com/moosetechnology/Moose/releases/download/v11.x.x/Moose11-stable-Pharo64-11.zip"
       ;;
     "Moose64-12"*)
-      echo "https://github.com/moosetechnology/Moose/releases/download/continuous/Moose12-development-Pharo64-12.zip"
+      echo "https://github.com/moosetechnology/Moose/releases/download/v12.0.0/Moose12-stable-Pharo64-12.zip"
```

Comment:
Will there ever be a 12.0.1 or so?

Reply:
Ah, yes, I should have used v12.x.x.
There will most probably not have a minor version update.
Moose major versions change approximately each year.
It is mostly a cluster of different projects and the finer versioning is managed at their level.
If by any chance we need a minor version change, I'll correct the version to v12.x.x.